<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>capture</class>
 <widget class="QWidget" name="capture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-2</y>
     <width>801</width>
     <height>482</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>801</width>
     <height>482</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>801</width>
     <height>482</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#frame{background-image: url(:/aaa.jpg);}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="videoLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgb(85, 87, 83);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="startCaptureButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>440</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>开始采集摄像头数据</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="return_2">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-2</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>111</width>
      <height>101</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>111</width>
      <height>101</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-image: url(:/image/return.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>160</y>
      <width>81</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="checkBox1">
       <property name="styleSheet">
        <string notr="true">QRadioButton {
	color: blue;
}
QRadioButton:indicator {
	width: 30; 
	height: 30;
}</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="checkBox2">
       <property name="styleSheet">
        <string notr="true">QRadioButton {
	color: blue;
}
QRadioButton:indicator {
	width: 30; 
	height: 30;
}</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>380</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="text">
     <string>检测</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
